pkgs/misc/uboot/default.nix

```diff
@@ -21,6 +21,7 @@
 , armTrustedFirmwareRK3399
 , armTrustedFirmwareS905
 , buildPackages
+, pkgs
```

Comment:
Missed that.

We generally do not use `pkgs` here. I see why it was needed (for `ubootRockchipTools`), but that would be solved by declaring it inside the `let` block instead, if it was still needed.